colcon-test/README.md

```diff
@@ -0,0 +1,83 @@
+# colcon-test
+
+This action runs `colcon test` with label.
+Note that you need to build target packages before running this action.
+
+## Usage
+
+```yaml
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: ros:galactic
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Get modified packages
+        id: get-modified-packages
+        uses: autowarefoundation/autoware-github-actions/get-modified-packages@tier4/proposal
+
+      - name: Build and test
+        if: ${{ steps.get-modified-packages.outputs.modified-packages != '' }}
+        uses: autowarefoundation/autoware-github-actions/colcon-build@tier4/proposal
+        with:
+          rosdistro: galactic
+          target-packages: ${{ steps.get-modified-packages.outputs.modified-packages }}
+          build-depends-repos: build_depends.repos
+
+  test:
+    needs: build-and-test
+    runs-on: ubuntu-latest
+    container: ros:galactic
+    strategy:
+      matrix:
+        label: [gtest, launch_test]
```

Comment:
[IMO] This style is better since it's easier(less diff) when updating the label.

```suggestion
        label: 
          - gtest
          - launch_test
```